### ISSUE TITLE
Publish SCM / URL Information in Plugin POM

### ISSUE THREAD

**Issue by austinarbor** [gradle/plugin-portal-requests#214]
### Expected Behavior

The POM files for plugin artifacts should include the scm/project URL

### Current Behavior (optional)

The URLs are not included in the POM

### Context

Currently, Dependabot updates for Gradle plugins in GitHub do not include any of the release note information for the update.

If you look at the [Dependabot source code](https://github.com/dependabot/dependabot-core/blob/main/gradle/lib/dependabot/gradle/metadata_finder.rb#L59-L63) it appears to try and look up the project, scm, and issue management URLs from the POM in order to track the project back to GitHub and link the release notes to it.

Having the release note information available directly in the Dependabot PR body would be a huge QOL improvement

**Comment by ljacomet** [comment]
Currently, [marker POMs for Gradle plugins](https://docs.gradle.org/current/userguide/plugins.html#sec:plugin_markers) are generated on the server side when served from the Gradle Plugin Portal. This decision was made long ago and would be quite complicated to change.

We would need to consider the implications of changing what those files contain or decide to include that information only after a certain date.

**Comment by ljacomet** [comment]
Closing this as a duplicate of:
* #173
